src/entries/popup/hooks/useInfiniteTransactionList.ts

```diff
@@ -54,56 +55,13 @@ export default function ({
     isInitialLoading,
     refetch,
     status,
-  } = useConsolidatedTransactions(
-    { address, currency },
-    {
-      select: (data) => {
-        const selectedPages = data.pages.map((page) => {
-          return {
-            ...page,
-            transactions: page.transactions.filter((transaction) =>
-              chains.map((chain) => chain.id).includes(transaction.chainId),
-            ),
-          };
-        });
-        return {
-          ...data,
-          pages: selectedPages,
-        };
-      },
-      onSuccess: (data) => {
-        if (data?.pages) {
-          const latestTransactions = data.pages
-            .map((p) => p.transactions)
-            .flat()
-            .filter((t) => isLowerCaseMatch(t.from, address))
-            .reduce(
-              (latestTxMap, currentTx) => {
-                const currentChain = currentTx?.chainId;
-                if (currentChain) {
-                  const latestTx = latestTxMap.get(currentChain);
-                  if (!latestTx) {
-                    latestTxMap.set(currentChain, currentTx);
-                  }
-                }
-                return latestTxMap;
-              },
-              new Map(
-                getSupportedChainIds().map((chain) => [
-                  chain,
-                  null as RainbowTransaction | null,
-                ]),
-              ),
-            );
-          watchForPendingTransactionsReportedByRainbowBackend({
```

Comment:
moved this to `useTransactionListForPendingTxs`

i had a problem with this hook being used to render txs

- hook was pinging backend with all chains supported, since we're also polling be to get txs in hidden networks
- this was making BE to return pagination data for chains that the user could have hidden, making the list look off
- this was even more noticeable when you turned off all networks but you had one custom network enabled, BE would be giving the extension 100s transactions with many pages, even tho we weren't rendering the whole list making it 💥 

so i put `useTransactionListForPendingTxs` in App, to update pending txs with all the txs BE has